### ISSUE TITLE
TypeHintDeclaration.UselessDocComment: False positive when description contains dollar/variable

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#241]
```php
class Foo
{
    /**
     * @param bool $foo describes something about $bar
     */
    public static function rules(bool $foo, int $bar) : void
    {
    }
}
```
Produces:
> Method \Foo::rules() does not need documentation comment.

Appears as UselessDocComment or UselessParameterAnnotation. This is related to the variable in description - once removed, error disappears.

Side effect: Fixer removes the comment for the parameter entirely.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/a0fb36df3dd72837971f6b96d77c69c84d6a84f7
